src/types/subscription.ts

```diff
@@ -37,3 +37,12 @@ export const projectSubscriptionMethods = [
     label: "Create a JIRA Issue",
   },
 ];
+
+// Map a NotificationMethods enum to its label
+export const notificationMethodToCopy = projectSubscriptionMethods.reduce(
```

Comment:
nit. Could this use jsdoc syntax so we can get IntelliSense previews?

Reply:
Lmk if there's something else you were looking for besides using `/** */` syntax, I wasn't exactly sure since this isn't a function!

Reply:
I wonder if we should look into an ES Lint plugin to enforce jsdoc! There's this: https://www.npmjs.com/package/eslint-plugin-jsdoc

Reply:
I like this i'll add it to the UI round table!